composer.json

```
@@ -22,6 +22,7 @@
     "require-dev": {
         "doctrine/coding-standard": "^11.1",
         "phpbench/phpbench": "^1.2",
+        "psr/log": "^1.1.4",
```

The versions 2 and 3 added respectively parameters and return types.
```suggestion
        "psr/log": "^1.1.4|^2|^3",
```

---

We may mark the dependency as "compatible" in composer, one day. https://github.com/composer/composer/issues/11635

---

Resolving this, as `|^2|^3` has been added.